src/components/UserSettings.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+import { useState } from "react";
+import EditableField from "./EditableField";
+import { text } from "stream/consumers";
+
+function UserSettings() {
+  const [fullName, setFullName] = useState("Diana Damian");
+  const [email, setEmail] = useState("diana.damian@gmail.com");
+  return (
+    <div className="w-full h-full flex">
+      <div className="m-2 p-1 border w-full h-3/5">
+        <h2 className="text-lg font-thin m-2"> ACCOUNT DETAILS </h2>
+        <div className="p-1">
+          <EditableField
+            label={"Full Name"}
```

Comment:
No need to put the `{` if it's just a string. It can be simply `label="Full Name"`.

Do the same for all